src/UI/Controllers/ExpenseReportController.cs

```diff
@@ -40,7 +40,7 @@ public ActionResult Manage(string id, EditMode mode)
 
             if (mode == EditMode.New)
             {
-                expenseReport = _expenseReportBuilder.Build(currentUser);
+                expenseReport = _expenseReportBuilder.Build(currentUser).Result;
```

Comment:
Avoid calling `Task<T>.Result` as this can cause deadlocks in your code.  Stephen Cleary explains the scenario very well [here](https://blog.stephencleary.com/2012/07/dont-block-on-async-code.html) and [here](https://blog.stephencleary.com/2014/12/a-tour-of-task-part-6-results.html).

You will want to instead propagate the change to async all the way through to the framework's controller method, which will need to be async.

Reply:
Right, that was a mistake, I wasn't trying to merge the branches. I was trying to update my own branch. So this commit needs to be undone.

Reply:
Great, thanks.